### DISCUSSION TITLE
Public sessions

### DISCUSSION THREAD

**Discussion by tedr56** [Budibase/budibase#11659] [Help]
Hi all.
I'm trying to make an quick exchange app wich show the prices and price difference between two people.
I'm still at the beginning of the process but I'm wondering if all works well, can I make it with public sessions?
Like if two other people want to exchange too and don't have to wait the first two to finish.
And since it's a quick helper app, I have no intention to make people register and login for that.
Thanks in advance for any insight.
And thanks to all budibase devs.

**Comment by melohagan** [comment]
> price difference between two people

What kind of app is this❓

**Reply by tedr56** [reply]
Just an helper app to try to make exchanges even.

**Reply by melohagan** [reply]
To be honest I don't follow the use-case

**Reply by tedr56** [reply]
If you want to exchange items with someone. You'll have to know every items worth for both of you. If you have like two items each, that's easy, but if it's more like 30, it can be challenging to keep track. My app tells the price difference between items of both.
